green/command.py

```diff
@@ -3,7 +3,8 @@
 try:
     from setuptools import Command
 except ImportError:  # pragma: no cover
-    from distutils.cmd import Command
+    # The distutils package is deprecated as of https://peps.python.org/pep-0632/
+    from distutils.cmd import Command  # type: ignore
```

Comment:
Let's try to remove it, then.

If this breaks any of the versions we still support, then lets modify the comment to make it clear which version(s) still require this.

Reply:
This PR adds setuptools to the requirements anyways, so the distutils fallback is more than redundant now.